src/pages/journal/journal.ts

```diff
@@ -110,15 +110,16 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
     this.toast.present();
   };
 
-  private createSlots = (slotData: any) => {
-    if (slotData) {
+  private createSlots = (emission: any) => {
+    if (Array.isArray(emission) && emission.length > 0) {
```

Comment:
can we invert this if to reduce nesting ?

Reply:
Done.